tests/tekton-resources/tasks/generators/clusterloader/sustained-scheduler-throughput.yaml

```diff
@@ -0,0 +1,111 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sustained-scheduler-throughput
+  namespace: scalability
+spec:
+  description: "clusterloader2 task to run various types of cl2 tests on a given cluster."
+  params:
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/dheeraj-coding/perf-tests.git
+  - name: cl2-branch
+    description: "The branch of clusterloader2 you want to use"
+    default: "steady-load"
+  - name: cl2-scheduler-throughput-pods
+    default: 500000
+  - name: cl2-scheduler-throughput-threshold
+    description: "scheduler throughput threshold"
+    default: "150"
+  - name: cl2-scheduler-steady-qps
+    description: "pods scheduler steady qps"
+    default: "170"
+  - name: cl2-scheduler-create-qps
+    default: "1700"
+  - name: nodes
+    description: "number of dataplane nodes to run the load test against"
+    default: "1000"
```

Comment:
pod density seems high for default pods count that you have.

Have you tested with these defaults ?

Reply:
The number of pods are high. I tested with much lower values of upto 80k pods in a 1000 node cluster. Intend to run these test in 5k clusters and hence changed the new default to 200k

Reply:
Are you sure these pods actually came into running state when you had 80K pods created on 1k node cluster ?